engine/src/main/java/org/terasology/logic/console/commands/ClientCommands.java

```diff
@@ -68,6 +73,22 @@ public String setWorldTime(@CommandParam("day") float day) {
         return "World time changed";
     }
 
+    /**
+     * Permanently halts the sun's position and angle
+     * @param day Float containing day to be set
+     * @return String message containing message to notify user
+     */
+    @Command(shortDescription = "Permanently halts the sun's position and angle", requiredPermission = PermissionManager.CHEAT_PERMISSION)
+    public  String setSunPosition(@CommandParam("day") float day) {
+        worldProvider.getTime().setDays(day);
```

Comment:
So the `worldTime` keeps on incrementing normally after setting it to `days`, only the sun remains fixed.

Reply:
What is the impact of not setting the world time?  It seems there is a setWorldTime command already for this purpose,  I am thinking we should probably keep each command isolated to its sole task.  Maybe I am missing something though,  thoughts?

Reply:
I had it earlier with this command so that the `worldTime` matches with the sun's position. But it doesn't make any difference to the original purpose of this command, so I have removed it :smile: